blankiety_galantow/resources/game/css/main.css

```diff
@@ -159,6 +159,11 @@
 #chat-input {
     width: 100%;
 }
+#stop-meme {
+    padding: 0.5em;
+    max-width:100%;
+    max-height:100%;
+}
```

Comment:
Jeśli nie ma obrazka, to chyba niepotrzebne? Chyba, że zostawimy z nadzieją, że znajdziemy jakiś fajny obrazek :-)
```suggestion
```